rotkehlchen/chain/ethereum/modules/makerdao/vaults.py

```diff
@@ -650,7 +650,7 @@ def _get_vaults_of_address(
 
         vaults = []
         for idx, identifier in enumerate(result[0]):
-            urn = to_checksum_address(result[1][idx])
+            urn = deserialize_ethereum_address(result[1][idx])
```

Comment:
exact same thing as above here. DeserializationError is not handled anywhere above in the stack, but `RemoteError` is.